backend/README.md

```diff
@@ -28,10 +28,17 @@ psql counterfactualcovid -c "CREATE USER django WITH PASSWORD 'value';" # where
 You can test that `PostgreSQL` is correctly configured by running
 
 ```bash
-psql counterfactualcovid -c "SELECT * FROM pg_user;"
+psql counterfactualcovid -U django -c "SELECT * FROM pg_user;"
 ```
 
-and checking that you get a list of users back.
+and checking that you get a list of users back. If you get an error here is because the user `django` doesn't exist. To solve this just create a new user with:
+
+```bash
+createuser django -P
```

Comment:
You mean this line @jemrobinson?

Reply:
Yes. I think that's nicer than having to run a command inside `psql`.